src/app/static/src/app/components/plots/bar/utils.ts

```diff
@@ -17,9 +16,11 @@ export interface ErrorBars {
     }
 }
 
-export type ChartDataSetsWithErrors =  ChartDataset & {
-    errorBars?: ErrorBars
-}
+type ChartDataSetsWithErrors<Data = BarChartDefaultData> = BarChartDataset<Data> & { errorBars?: ErrorBars }
+
+type ChartDataWithErrors<Data = BarChartDefaultData> = Omit<ChartData<"bar", Data, string>, "datasets"> & { datasets: ChartDataSetsWithErrors<Data>[] }
+
+export interface BarChartData<Data = BarChartDefaultData> extends ChartDataWithErrors<Data> { maxValuePlusError: number }
```

Comment:
this is same as before but now with our `ChartDataWithErrors` type. It also has the `BarChartDefaultData` type which is what `chart-js` uses as the default for the data generic